swap_meet/electronics.py

```diff
@@ -1,2 +1,10 @@
-class Electronics:
-    pass
+from .item import Item
+
+class Electronics(Item): # Updated class to inherit from Item
+    def __init__(self, id=None, type="Unknown", condition=0):
+        super().__init__(id, condition)  # pass id and condition to Item, move 
+                                        # the line above child's custom attribute
+        self.type = "Unknown" if type == "Unknown" else type
```

Comment:
Like my comment in `Decor`, we should invert this logic so that the attribute is set to `type` if `type` has a non-default value and then the `else` can set it to "Unknown".

This logic is inverted from what you have on line 7 in `Clothing`. So it would be good that this logic matches what you already have for the sake of consistency too.

```suggestion
        self.type = type if type != "Unknown" else = "Unknown"
```

Reply:
I'll update the code to reflect that. Thanks for pointing it out!